feat(changelog): messaging-and-queuing-changed-increased-messaging-resou 2024-04-25 (#3115)
scaleway/docs-content: changelog/april2024/2024-04-25-messaging-and-queuing-changed-increased-messaging-resou.mdx

---
title: Increased Messaging Resources Quotas
status: changed
author:
    fullname: 'Join the #messaging-queuing channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-04-25
category: serverless
product: messaging-and-queuing
---

> Messaging ans Queuing protocols  are now scalable up to 50 resources (SQS queues, SNS topics & NATS streams) per Scaleway project.
> 
> More info on Messaging and Queuing limitations here :https://www.scaleway.com/en/docs/serverless/messaging/reference-content/limitations/

Reviewer: ```suggestion
The messaging and queuing protocols on Scaleway are evolving. Now, you can scale up to 50 resources, including SQS queues, SNS topics, and NATS streams, per Scaleway project.

This update empowers you to handle larger workloads and enables more robust communication within your projects. 

For detailed information on the limitations and capabilities of our Messaging and Queuing protocols, please refer to our [Messaging and Queing documentation](https://www.scaleway.com/en/docs/serverless/messaging/reference-content/limitations/).
```

---

> title: Increased Messaging Resources Quotas

Reviewer 2: ```suggestion
title: Increased resource quotas
```

status: changed
author:
    fullname: 'Join the #messaging-queuing channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-04-25
category: serverless
product: messaging-and-queuing
---

Scaleway Messaging and Queuing is evolving. We have increased the limitations on our resource quotas, meaning that each Scaleway Project can now scale up to:
- 50 SQS queues
- 50 SNS topics
- 50 NATS streams

This update empowers you to handle larger workloads and enables more robust communication.

> For detailed information on the limitations and capabilities of our Messaging and Queuing protocols, please refer to the [Messaging and Queuing documentation](https://www.scaleway.com/en/docs/serverless/messaging/reference-content/limitations/).

Reviewer 3: ```suggestion
For detailed information on the limitations and capabilities of our Messaging and Queuing protocols, please refer to the [Messaging and Queuing documentation](/serverless/messaging/reference-content/limitations/).
```
